.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - '6'
-  - '5'
-  - '4'
+  - '13'
+  - '12'
+  - '10'
 before_install:
   - 'npm install -g npm@latest'
```

Comment:
@cossssmin I think this is not necessary

Reply:
OK, will not specify any Node version then.

What about linting still failing? Can we just disable those two rules, or...?

Reply:
I think specifing node versions are important. otherwise we wont be able to know in which version it will fail. 
I think @Scrum meant with `npm install -g npm@latest` thing, travis does that out of the box

> What about linting still failing? Can we just disable those two rules, or...?

can you fix them ?

Reply:
So this'll just use Node `0.10.48`, as you can see in the failing build. Did you mean to use only 10? Or...?

Reply:
I fixed the config for Travis, now I need to fix the tests

Reply:
Thanks Ivan, got confused by that diff 🤷‍♂️

Reply:
Do not worry, we appreciate any contribution))